samples/GSC/array_shared.gsc

```diff
@@ -0,0 +1,1321 @@
+#using scripts\codescripts\struct;

```

Comment:
This file is massive and I don't think it adds much more than the sample above. Please remove it.